i3/2-yajl.md

```diff
@@ -0,0 +1,54 @@
+# Introduction to yajl-2.1.0
+Yajl is yet another json library.
+
+## Package Information
+- Download (HTTP): https://github.com/lloyd/yajl/archive/refs/tags/2.1.0.tar.gz
+- Download md5sum: 6887e0ed7479d2549761a4d284d3ecb0
+- Download Size: 84K
+- Build Size: 2.3M
+- Build Time: < 0.1 SBU
+
+## Additional Downloads
+- Required patch: https://gitlab.archlinux.org/archlinux/packaging/packages/yajl/-/raw/main/yajl-2.1.0-CVE-2017-16516.patch
+- Required patch: https://gitlab.archlinux.org/archlinux/packaging/packages/yajl/-/raw/main/yajl-2.1.0-CVE-2022-24795.patch
+- Required patch: https://gitlab.archlinux.org/archlinux/packaging/packages/yajl/-/raw/main/yajl-2.1.0-memory_leak.patch
+
+## yajl Dependencies
+### Required
+  [cmake](https://www.linuxfromscratch.org/blfs/view/svn/general/cmake.html)
+  [glibc](https://www.linuxfromscratch.org/lfs/view/development/chapter08/glibc.html)
```

Comment:
Mentioning glibc is unnecessary. LFS packages should be a baseline and its packages should not be listed as dependencies in this project and LFS books besides LFS itself.